site/app/views/grading/ElectronicGraderView.php

```diff
@@ -891,6 +891,159 @@ public function randomizeButtonWarning(Gradeable $gradeable) {
         ]);
     }
 
+    public function hwGradingPopup(Gradeable $gradeable, GradedGradeable $graded_gradeable, string $type, int $display_version, bool $show_hidden_cases, bool $can_inquiry, bool $can_verify, bool $show_verify_all, bool $show_silent_edit, $rollbackSubmission, $from, array $solution_ta_notes, array $submitter_itempool_map, $anon_mode, $blind_grading, $window_id) {
```

Comment:
The list of parameters is quite long here.  It would be better to break it such that each parameter is on a separate line.  In addition, a return type should be added.